variables.tf

```diff
@@ -136,4 +136,10 @@ variable capacity_provider_strategies {
   capacity_provider - Short name of the capacity provider
   weight - Relative percentage of the total number of launched tasks that should use the specified capacity provider
   EOF
+}
+
+variable ephemeral_storage {
+  description = "The total amount, in GiB, of ephemeral storage to set for the task. The minimum supported value is 21 GiB and the maximum supported value is 200 GiB."
+  type    = number
+  default = 20
```

Comment:
If the minimum supported value, according to the description is 21 GiB, then we are we setting the default to 20?

Reply:
That is odd, I copied that from the AWS docs, but the default value is 20 in AWS :thonk:

I'll just change the text to say 20 I guess is the minimum